Foundation/CMakeLists.txt

```diff
@@ -97,7 +97,10 @@ set_target_properties(Foundation
 	DEFINE_SYMBOL Foundation_EXPORTS
 	)
 
-target_link_libraries(Foundation PUBLIC ${PCRE_LIBRARIES} ${ZLIB_LIBRARIES})
+if (POCO_UNBUNDLED)
+    target_link_libraries(Foundation PUBLIC ${PCRE_NAMES} ZLIB::ZLIB)
```

Comment:
Using `PCRE_NAMES` here seems very hacky. It's basically equivalent to just putting `pcre`.

I would prefer to change the `FindPCRE.cmake` module to provide an exported target that can be used.

Reply:
Hi @bachp,
Thanks for your comments !

I'm also not really happy with my solution for pcre but I didn't find a simpler solution for this. My first idea was also to modify `FindPCRE.cmake` to provide an exported target, but this not enough:
The `FindPCRE.cmake` is available during the compilation, but it is not part of the installed files. This end up with pre-build looking for an exported target that can not be resolved. This can be solved by installing `FindPCRE.cmake`. The problem is not here with `zlib` because cmake have already a  `FindZLIB.cmake` module.

If installing a module to find PCRE as part of Poco is not an issue, I would be happy to push the required changes.

Reply:
I think installing the modified `FindPCRE.cmake` along Poco would be an acceptable solution. In my opinion you can continue.

Reply:
First, thanks @lmayencourt . I fail to see this issue when I refactor/cleaup all XXX_LIBRARIES places to use the modern art of cmake.
I agree with @bachp , because what with the "include path" for PCRE ? 
Let me cleanup the FindPCRE.cmake and install when POCO_UNBUNDLED is true.
I would like to use your merge request, is this ok for you @lmayencourt?

Finally, I found another place where `ZLIB_LIBRARIES` is used: PDF/CMakeList.txt:129 ....
I can also fix this.